uiowa/uiowa
# Miscellaneous areas of study fixes.

Resolves #3354

###  Testing
- blt sync --site admissions.uiowa.edu
- See Student Organizations link at https://admissions.local.drupal.uiowa.edu/academics/actuarial-science points to https://uiowa.campuslabs.com/engage instead of orgsync. For comparison see prod.
- Create a new AOS with no field data except required ones. Ensure all links in default text field are not broken.
- See extra spacing is removed from degree abbreviations: http://admissions.local.drupal.uiowa.edu/academics/business
- Four-Year Graduate Plan now reads as Four-Year Graduation Plan
- http://admissions.local.drupal.uiowa.edu/academics/groups is now listed alphabetically.
